src/where.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import {broadcast, getBroadcastShape} from './lib/broadcast.js';
+import {Tensor, sizeOfShape, Scalar} from './lib/tensor.js';
+
+/**
+ * Select the values from the trueValues or the other tensor depending on
+ * the corresponding Boolean values of the condition tensor.
+ * @param {Tensor} condition
+ * @param {Tensor} trueValues
+ * @param {Tensor} falseValues
+ * @return {Tensor}
+ */
+export function where(condition, trueValues, falseValues) {
+  const trueValuesReshape = trueValues.shape.length === 0 ?
+    new Scalar(trueValues.data) :trueValues;
```

Comment:
```suggestion
    new Scalar(trueValues.data) : trueValues;
```
When you touch this file in the future.